nessie/sql_templates/update_rds_indexes_student_profiles.template.sql

```diff
@@ -51,14 +74,32 @@ ON CONFLICT (sid) DO UPDATE SET
   sid=EXCLUDED.sid, uid=EXCLUDED.uid, first_name=EXCLUDED.first_name, last_name=EXCLUDED.last_name,
   level=EXCLUDED.level, gpa=EXCLUDED.gpa, units=EXCLUDED.units, transfer=EXCLUDED.transfer,
   expected_grad_term=EXCLUDED.expected_grad_term, terms_in_attendance=EXCLUDED.terms_in_attendance,
-  hist_enr=FALSE;
+  hist_enr=EXCLUDED.level;
```

Comment:
@pauline2k Does this work because hist_enr students don't have a level?

Reply:
No, it doesn't work because I copied and pasted like a fool. One sec!

Reply:
Amended.

Reply:
Ah! Thanks